index.html

```diff
@@ -59,7 +63,136 @@ <h1 class="info">Hey There. <br>
                </ul>                            
            </div>                   
         </section>           
-    </div>            
+        
+    </div>     
+
+    
+    <div class="wrapper-works">
+        <header class="title">
+            My Recent Works
+        </header>
+
+        <div class="line"></div>
+
+        <div class="image-work">  </div>
```

Comment:
It looks great! but please make sure to replace the `background-image` with the `img` tag here because it's not an illustration.